datalad/support/gitrepo.py

```diff
@@ -697,7 +710,7 @@ def __eq__(self, obj):
     @classmethod
     def is_valid_repo(cls, path):
         """Returns if a given path points to a git repository"""
-        return exists(opj(path, '.git', 'objects'))
+        return exists(opj(path, '.git'))
 
```

Comment:
If we need a more extensive check of the validity of a repo, I'd vote for having two methods.
This one usually is used to find out, whether we should view a directory as a repository.
This should be cheap and not too sophisticated, I think.

If we say that just a `.git` isn't sufficient, one could also argue, that `.git/objects` isn't, since the sheer existence doesn't mean its content is valid ...